backend/src/main/java/gov/cdc/usds/simplereport/api/apiuser/ApiUserResolver.java

```diff
@@ -22,15 +28,35 @@
 	private ApiUserService _userService;
 	private OrganizationService _organizationService;
 
-	public ApiUserResolver(OrganizationService os, ApiUserService users, OrganizationInitializingService initer) {
+	public ApiUserResolver(OrganizationService os, ApiUserService users) {
 		_organizationService = os;
 		_userService = users;
 	}
 
 	public User getWhoami() {
 		ApiUser currentUser = _userService.getCurrentUser();
-		Optional<CurrentOrganizationRoles> currentOrgRoles = _organizationService.getCurrentOrganizationRoles();
-		Boolean isAdmin = _userService.isAdminUser(currentUser);
+		Optional<OrganizationRoles> currentOrgRoles = _organizationService.getCurrentOrganizationRoles();
+		Boolean isAdmin = _userService.isAdmin(currentUser);
         return new User(currentUser, currentOrgRoles, isAdmin);
 	}
+
+	public List<User> getUsers() {
+		Organization org = _organizationService.getCurrentOrganization();
+		List<ApiUser> users = _organizationService.getUsersInCurrentOrg(OrganizationRole.USER);
```

Comment:
Would this only get users with the role USER, or everyone? From a quick scan, the `OrganizationRole.USER` parameter is used to query the okta group -- are only users with role USER in that group?

I ask because i assume the graphql query for `users` would return all humans, not just USER humans.

Reply:
it gets everyone. everyone in an organization is in the USER Okta group for that org. what the lines after that do is assign the specific roles of all users based on the additional Okta groups (ADMIN, ENTRY_ONLY) they are in for that organization.